post_installation_configuration/coreos-layering.adoc

```diff
@@ -40,31 +57,78 @@ Some Hotfixes require a Red Hat Support Exception and are outside of the normal
 +
-.Example Containerfile to apply a Hotfix
+.Example on-cluster Containerfile to apply a Hotfix
 [source,yaml]
 ----
 # Using a 4.12.0 image
-FROM quay.io/openshift-release-dev/ocp-release@sha256...
-#Install hotfix rpm
-RUN rpm-ostree override replace https://example.com/myrepo/haproxy-1.0.16-5.el8.src.rpm && \
-    rpm-ostree cleanup -m && \
-    ostree container commit
+containerfileArch: noarch
+content: |-
+  FROM configs AS final
+  #Install hotfix rpm
+  RUN rpm-ostree override replace https://example.com/myrepo/haproxy-1.0.16-5.el8.src.rpm && \
+      rpm-ostree cleanup -m && \
+      ostree container commit
+----
++
+.Example out-of-cluster Containerfile to apply a Hotfix
+[source,yaml]
+----
+# Using a 4.12.0 image
+containerfileArch: noarch
+content: |-
+  FROM quay.io/openshift-release-dev/ocp-release@sha256...
+  #Install hotfix rpm
+  RUN rpm-ostree override replace https://example.com/myrepo/haproxy-1.0.16-5.el8.src.rpm && \
+      rpm-ostree cleanup -m && \
+      ostree container commit
 ----
 
 * *{op-system-base} packages*. You can download {op-system-base-full} packages from the link:https://access.redhat.com/downloads/content/479/ver=/rhel---9/9.1/x86_64/packages[Red Hat Customer Portal], such as chrony, firewalld, and iputils.
 +
-.Example Containerfile to apply the firewalld utility
+.Example on-custer Containerfile to apply the firewalld utility
+[source,yaml]
+----
+containerfileArch: noarch
+content: |-
+  FROM configs AS final
+  ADD configure-firewall-playbook.yml .
+  RUN rpm-ostree install firewalld ansible && \
+      ansible-playbook configure-firewall-playbook.yml && \
+      rpm -e ansible && \
+      ostree container commit
+----
++
+.Example out-of-cluster Containerfile to apply the firewalld utility
+[source,yaml]
+----
+containerfileArch: noarch
+content: |-
+  FROM quay.io/openshift-release-dev/ocp-release@sha256...
+  ADD configure-firewall-playbook.yml .
+  RUN rpm-ostree install firewalld ansible && \
+      ansible-playbook configure-firewall-playbook.yml && \
+      rpm -e ansible && \
+      ostree container commit
+----
++
+.Example on-cluster Containerfile to apply the libreswan utility
 [source,yaml]
 ----
-FROM quay.io/openshift-release-dev/ocp-release@sha256...
-ADD configure-firewall-playbook.yml .
-RUN rpm-ostree install firewalld ansible && \
-    ansible-playbook configure-firewall-playbook.yml && \
-    rpm -e ansible && \
+# Get RHCOS base image of target cluster `oc adm release info --image-for rhel-coreos`
```

Comment:
issue: They wouldn't need to get the base image pullspec if they're doing this on-cluster since the on-cluster layering process will inject it for them. All they need to provide is the `FROM configs AS final` line.